### PR TITLE
Addd relation between applicationSeason and application

### PR DESCRIPTION
**Pull request by s0lvang**

### CODE REVIEW

File: backend/__tests__/models/test_application_season_model.py [hunk 1]
```diff
@@ -26,6 +29,38 @@ def test_no_submit_on_wrong_data(db_session):
...
+def test_cascading(db_session):
+
+    start = datetime.now()
+    end = datetime.now() + timedelta(days=180)
+    applicationStart = start + timedelta(days=7)
+    applicationEnd = start + timedelta(days=21)
+    season = ApplicationSeason(start, end, applicationStart, applicationEnd)
+
+    user = User(username="yoo", sub="sub", email="email", fullname="Dudeman")
+    application = Application(
```
Comment (PeterRydberg) [inline backend/__tests__/models/test_application_season_model.py]: Could this use the createApplication function?

File: backend/__tests__/integration_tests/test_application.py [hunk 2]
```diff
@@ -25,6 +26,7 @@ def setUp(self):
 
     @mock_authentication_context
     def test_new_application_without_partner(self):
+        createBasicSeason(db.session)
```
Comment (PeterRydberg) [inline backend/__tests__/integration_tests/test_application.py]: Is this used for anything? The test doesn't expect to receive a connected season, but that might be intended behaviour.

**Comment by PeterRydberg** [comment]
Good job! Mainly wondering if the application is supposed to return with its season, as it might be relevant for rollover seats, but it might not be necessary now.

File: backend/__tests__/integration_tests/test_application.py (hunk 2 above)
Comment (s0lvang) [reply backend/__tests__/integration_tests/test_application.py]: The applicationService fetches getCurrentOrNext so a applicationSeason is required.

File: backend/__tests__/integration_tests/test_application.py [hunk 3]
```diff
@@ -64,6 +66,7 @@ def test_new_application_without_partner(self):
 
     @mock_authentication_context
     def test_new_application_without_existing_partner(self):
+        createBasicSeason(db.session)
```
[the remark this answers is not in the document]
Comment (s0lvang) [reply backend/__tests__/integration_tests/test_application.py]: #SAME AS ABOVE

**Pull request merged**